document-store/src/main/java/org/hypertrace/core/documentstore/parser/GroupByAliasGetter.java

```diff
@@ -0,0 +1,18 @@
+package org.hypertrace.core.documentstore.parser;
+
+import org.hypertrace.core.documentstore.expression.impl.FunctionExpression;
+import org.hypertrace.core.documentstore.expression.impl.IdentifierExpression;
+
+@SuppressWarnings("unchecked")
+public class GroupByAliasGetter implements GroupTypeExpressionVisitor {
+
+  @Override
+  public String visit(FunctionExpression expression) {
+    return null;
```

Comment:
Better to throw exception instead of null.
Or, alternatively return optional of string to avoid NullPointerExceptions.